src/Features/OnboardingTasks/Tasks/Appearance.php

```diff
@@ -34,6 +35,12 @@ public static function get_task() {
 			'is_complete'  => Task::is_task_actioned( 'appearance' ),
 			'can_view'     => true,
 			'time'         => __( '2 minutes', 'woocommerce-admin' ),
+			'extra'        => array(
```

Comment:
I tend to agree with this.  I chose `extra` to provide some distinction between `additionalInfo`, but it may be more confusing than just `additionalData` so I reverted to this.